include/nuttx/fs/fs.h

```diff
@@ -474,10 +483,6 @@ struct file
   uint8_t           f_tag_fdcheck; /* File owner fdcheck tag, init to 0 */
 #endif
 
-#if CONFIG_FS_BACKTRACE > 0
-  FAR void         *f_backtrace[CONFIG_FS_BACKTRACE]; /* Backtrace to while file opens */
```

Comment:
need keep  in `struct fd`

Reply:
I thought it needs to be in struct file to track where the open() calls come from, but I might have not understood this feature.

Can you explain why, what is this backtrace for and how it's supposed to work ?